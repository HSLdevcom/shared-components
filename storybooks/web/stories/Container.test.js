import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  configPath: 'storybook_web',
  storyKindRegex: /^Container$/
});
