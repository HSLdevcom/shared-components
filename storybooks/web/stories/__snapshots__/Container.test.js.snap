// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Container default 1`] = `
<div
  className="sc-kvZOFW kZFdYp"
>
  <p>
    Some content
  </p>
</div>
`;
